test/all-templates.test.js

```diff
@@ -309,15 +269,34 @@ describe('CI template verification', () => {
           return;
         }
 
-        fs.readdir(functionsDir, (err, functions) => {
+        const functions = [];
```

Comment:
Looks like this wasn't checking for tests nested in a subdir, and since the functions and tests I added are nested it was failing here so added support for that.